### PR TITLE
Diagnostics for CommitOnSave failures

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs [hunk 1]
```diff
@@ -141,6 +140,16 @@ public void SetUseSuggestionMode(bool value)
             }
         }
 
+        public void VerifyNotSaved()
```
Comment (JoeRobich) [inline src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs:143]: Would naming these methods `Assert*` make it more clear that they will throw?

Comment (sharwell) [reply src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs:143]: Not sure. Either way seems to work.

**Pull request merged**